### PR TITLE
Bug 1130250 - [Stingray][AppDeck] The focused button at bottom the should be a complete circle

### PR DESCRIPTION
**Pull request by sean2449**
...e a complete circle

### CODE REVIEW

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=64e21418d523)

**Comment by rexboy7** [comment]
Again, please change your commit message.
1. Use [Stingray][AppDeck] ....
2. Is there a better way to describe "the bottom app that you selected"?

File: tv_apps/app-deck/style/app_deck.css [hunk 1]
```diff
@@ -50,6 +50,12 @@ section {
   scroll-behavior: smooth;
 }
 
+#app-deck-grid-view:after {
+  content: "";
+  display: block;
+  height: 4rem;
```
Comment (huchengtw-moz) [inline tv_apps/app-deck/style/app_deck.css]: why 4 rem?

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=14c9cd588ef8)

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=add6031ddd82)

**Pull request merged**